modules/installation-osp-default-kuryr-deployment.adoc

```diff
@@ -0,0 +1,295 @@
+--output-env-file=/home/stack/templates/overcloud_images.yaml \
+--output-images-file /home/stack/local_registry_images.yaml
+----
+
+Verify that the created *_local_registry_images.yaml_* contains the Octavia images.
+
+.Octavia images in local registry file
+----
+...
+- imagename: registry.access.redhat.com/rhosp13/openstack-octavia-api:13.0-43
+  push_destination: <local-ip-from-undercloud.conf>:8787
+- imagename: registry.access.redhat.com/rhosp13/openstack-octavia-health-manager:13.0-45
+  push_destination: <local-ip-from-undercloud.conf>:8787
+- imagename: registry.access.redhat.com/rhosp13/openstack-octavia-housekeeping:13.0-45
+  push_destination: <local-ip-from-undercloud.conf>:8787
+- imagename: registry.access.redhat.com/rhosp13/openstack-octavia-worker:13.0-44
+  push_destination: <local-ip-from-undercloud.conf>:8787
+----
+
+[NOTE]
+The versions of the Octavia containers will vary depending upon the specific
+{rh-openstack} release installed.
+
+
+The following step pulls the container images from registry.access.redhat.com
+to the undercloud node. This may take some time depending on the speed of the
+network and undercloud disk.
+
+----
+(undercloud) $ sudo openstack overcloud container image upload \
+  --config-file  /home/stack/local_registry_images.yaml \
+  --verbose
+----
+
+
+As an Octavia Load Balancer is used to access the OpenShift API, there is a
+need to increase their listeners default timeouts for the connections.
+The default timeout is 50 seconds. Increase the timeout to 20 minutes by
+passying the following file to the overcloud deploy command:
+
+----
+(undercloud) $ cat octavia_timeouts.yaml
+parameter_defaults:
+  OctaviaTimeoutClientData: 1200000
+  OctaviaTimeoutMemberData: 1200000
+----
+
+[NOTE]
+This is not needed from Red Hat OpenStack Platform 14 and onwards.
+
+Install or update your overcloud environment with Octavia:
+
+----
+openstack overcloud deploy --templates \
+.
+.
+.
+  -e /usr/share/openstack-tripleo-heat-templates/environments/services-docker/octavia.yaml \
+  -e octavia_timeouts.yaml
+.
+.
+.
+----
+
+[NOTE]
+The command above only includes the files associated with Octavia. This command
+will vary based upon your specifc installation of OpenStack. See the official
+OpenStack documentation for further information. For more information on
+customizing your Octavia installation, see
+https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html-single/networking_guide/#planning_your_octavia_deployment[installation
+of Octavia using Director].
+
+[NOTE]
+If Kuryr SDN is used, the overcloud installation requires the "trunk" extension
+to be enabled at Neutron. This is enabled by default on Director deployments.
+Use the openvswitch firewall instead of the default ovs-hybrid when the Neutron
+backend is ML2/OVS. There is no need for modifications if the backend is
+ML2/OVN.
+
+In order to be able to enforce Network Policies across services (i.e., when
```

Comment:
s/In order to be able to/To
Also avoid *i.e.*. Our guidelines say to avoid it.
Example first sentence:
>To enforce network policies across Services, like when traffic goes through the Octavia load balancer, you must ensure...